components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/ResourceReader.kt

```diff
@@ -22,20 +22,22 @@ suspend fun readResourceBytes(path: String): ByteArray = DefaultResourceReader.r
  * @return The URI string of the specified resource.
  */
 @InternalResourceApi
-fun getResourceUri(path: String): String = DefaultResourceReader.getUri(path)
+fun getResourceUri(path: String): String = getDefaultResourceReader().getUri(path)
 
-internal interface ResourceReader {
+@ExperimentalResourceApi
+interface ResourceReader {
     suspend fun read(path: String): ByteArray
     suspend fun readPart(path: String, offset: Long, size: Long): ByteArray
     fun getUri(path: String): String
 }
 
-internal expect fun getPlatformResourceReader(): ResourceReader
-
-internal val DefaultResourceReader = getPlatformResourceReader()
+@ExperimentalResourceApi
+expect fun getDefaultResourceReader(): ResourceReader
```

Comment:
Why do we need these changes?
I mean why it should be public?
Isn't it enough to have a public `LocalResourceReader` composition local?

Reply:
Users are able to get it from the Local provider, aren't they?